datasets/BEIS-Renewables-obligation-certificates-and-generation-monthly/main.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+# coding: utf-8
+# +
+from gssutils import *
+import pandas as pd
+import json
+
+scraper = Scraper(seed="info.json")
+scraper.distributions = [x for x in scraper.distributions if hasattr(x, "mediaType")]
+scraper
+
+#Add cubes class
+cubes = Cubes("info.json")
+
+#Add TransformTrace
+trace = TransformTrace()
+# -
+
+# Extract latest distribution  of the required dataset and datasetTitle
+distribution  = scraper.distribution(latest=True, title = lambda x:"Renewables obligation: certificates and generation (monthly)" in x)
+datasetTitle = distribution.title
+distribution
+
+# Extract all the tabs and its content from the spread sheet 
+tabs = distribution.as_databaker()
+
+# List out all the tab names to cross verify with spread sheet
+for tab in tabs:
+    print(tab.name)
+
+columns = ["Technology Group", "Generation Type", "Roc Per Mwh", "Period", "Qtr", "Month", "Element"]
+
+# +
+# Filtering the tabs which are required and start stage-1 transform
+tabs_i_want = ["Financial Year", "Quarter", "FY-only sites", "Month"]
+
+tabs = [x for x in tabs if x.name in tabs_i_want]
+for tab in tabs:
+    print(tab.name)
+    
+    trace.start(datasetTitle, tab, columns, distribution.downloadURL)
+    cell = tab.excel_ref("A1")
+    
+    generation_type = cell.shift(1, 5).fill(DOWN).is_not_blank().is_not_whitespace()
+    trace.Generation_Type("Defined from cell B6 below which is not blank")
+
+    roc_per_mwh = cell.shift(2, 5).fill(DOWN).is_not_blank().is_not_whitespace()
```

Comment:
```suggestion
    roc_per_mwh = cell.shift(2, 5).fill(DOWN) - remove
```

Reply:
Removed footer  and included blank and whitespace in roc_per_mwh column.